community/php7/APKBUILD

```diff
@@ -101,11 +100,10 @@ _build() {
 		--enable-exif=shared \
 		--enable-ftp=shared \
 		--with-gd=shared --with-webp-dir=shared --with-jpeg-dir=shared --with-png-dir=shared --with-xpm-dir=shared \
-		--with-freetype-dir=shared --enable-gd-native-ttf --enable-gd-jis-conv \
+		--with-freetype-dir=shared --enable-gd-native-ttf --disable-gd-jis-conv \
```

Comment:
Because it broke any non-latin font rendering. This flag for japan users only any nobody knows why it enabled be default. It's an old known bug.